Src/CSharpier.Tests/TestFiles/PropertyDeclaration/PropertyDeclarations.cst

```diff
@@ -54,7 +54,9 @@ public class ClassName
         set;
     }
 
-    public virtual ICollection<SomeObject> SomeLongNameThatForcesALineBreak { get; set; } =
+    public virtual ICollection<
+        SomeObject
```

Comment:
We're breaking too eagerly here. `public virtual ICollection<SomeObject>` fits in one line.

Hypothetically, even if ti didn't, I think the type arguments should be the last group we break.
```cs
public virtual
    ICollection<SomeLooooooooooooooooooooooooooongType>
        SomeLongNameThatForcesALineBreak { get; set; } =
            HashSet<SomeLooooooooooooooooooooooooooongType>
```

Reply:
This is currently similar to how prettier breaks the following typescript
```
  public SomeName: SomeLongGenericClassThatBreaks<
    SomeRandomType,
    SomeOtherRandomType
  > = new SomeLongGenericClassThatBreaks(someValue, someOtherValue);
```
Although I'm not sure how much I like how that typescript breaks. I created https://github.com/belav/csharpier/issues/243